### PR TITLE
adding prop "phone" to disply filter like on mobile even if you are on desktop

### PR DESCRIPTION
**Pull request by GabiDanciu**
I need a prop that renders filter-navigator like on mobile even if the resolution is desktop

https://category--joythestore.myvtex.com/womenswear/dresses

image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- adding `phone` type on prop `layout` to the `FilterNavigator`.
```
Comment (victorhmp) [inline CHANGELOG.md:10]:
```suggestion
### Added
- `phone` option to the `layout` prop  from `FilterNavigator` component.
```

**Review by victorhmp — APPROVED** [review]
LGTM

File: react/FilterNavigator.js [hunk 2]
```diff
@@ -94,8 +95,9 @@ const FilterNavigator = ({
 
   const mobileLayout =
     (isMobile && layout === LAYOUT_TYPES.responsive) ||
-    layout === LAYOUT_TYPES.mobile
-
+    layout === LAYOUT_TYPES.mobile ||
+    !isMobile && layout === LAYOUT_TYPES.phone || isMobile && layout === LAYOUT_TYPES.phone
```
Comment (thalytafabrine) [inline react/FilterNavigator.js:99]: I think you can simplify it like this:
```suggestion
    layout === LAYOUT_TYPES.phone
```

Comment (icazevedo) [reply react/FilterNavigator.js:99]: I agree! You should do that.

**Review by icazevedo — CHANGES_REQUESTED** [review]

File: react/FilterNavigator.js (hunk 2 above)
Comment (GabiDanciu) [reply react/FilterNavigator.js:99]: Done. Thanks for suggestions :D

**Review by thalytafabrine — APPROVED** [review]

**Review by icazevedo — CHANGES_REQUESTED** [review]
Could you please fix the lint issues you've introduced on you last commit? thanks

**Review by icazevedo — APPROVED** [review]

**Review by victorhmp — APPROVED** [review]

File: docs/README.md [hunk 3]
```diff
@@ -407,7 +407,7 @@ Renders a filter selector for the fetched results.
...
 | `categoryFiltersMode`  | `enum` | Whether the category filters should use the `href` attribute with the category pages' URLs (`href`) or not (`default`). By default, the filters use HTML divs with `role="link"`. You may change this behavior by setting this prop's value to `href`, thereby creating a link building to improve the SEO ranking of your category pages. | `default`  |
-| `layout`  | `Enum` | Whether the Filter Navigator layout should be responsive (`responsive`) or not (`desktop`). You may use `desktop` when the Filter Navigator was configured to be displayed in a [drawer](https://vtex.io/docs/components/content-blocks/vtex.store-drawer@0.9.0). | `responsive`  |
+| `layout`  | `Enum` | Whether the Filter Navigator layout should be responsive (`responsive`) or not (`desktop`, `phone`). You may use`phone` when mobile layout is needed on the desktop resolution and `desktop` when the Filter Navigator was configured to be displayed in a [drawer](https://vtex.io/docs/components/content-blocks/vtex.store-drawer@0.9.0). | `responsive`  |
```
Comment (igorbrasileiro) [inline docs/README.md:410]:
```suggestion
| `layout`  | `Enum` | Whether the Filter Navigator layout should be responsive (`responsive`) or not (`desktop`, `phone`). You may use `phone` when the mobile layout is needed on the desktop resolution and `desktop` when the Filter Navigator was configured to be displayed in a [drawer](https://vtex.io/docs/components/content-blocks/vtex.store-drawer@0.9.0). | `responsive`  |
```

**Review by igorbrasileiro — APPROVED** [review]

**Pull request merged**